scripts/report_errors.sh

```diff
@@ -44,6 +43,26 @@ function post_error_to_github
       --header 'content-type: application/json' \
       -u stackenbotten:$GITHUB_ACCESS_TOKEN \
       --data "{\"body\": \"There was an error when running \`$CIRCLE_JOB\` for commit \`$CIRCLE_SHA1\`:\n\`\`\`\n$FORMATTED_ERROR_MSG\n\`\`\`\nPlease check that your changes are working as intended.\"}"
+
+    post_review_comment_to_github
+}
+
+function post_review_comment_to_github {
+    GITHUB_API_URL="https://api.github.com/repos/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME/pulls/$CIRCLE_PR_NUMBER/comments"
+
+    sed -i 1d $ERROR_LOG
```

Comment:
What does this do?

Reply:
This uses the stream edit (sed) command to remove the first line from the error log in place (without creating a new file). It is done to prepare for further processing using grep, as the first line just announces the fact that there are errors. The next lines in the error log contains errors, one per line.